src/System.Private.Uri/src/System/UriHelper.cs

```diff
@@ -744,7 +744,7 @@ internal static bool IsNotSafeForUnescape(char ch)
             }
             return false;
         }
-        
+
         // "Reserved" and "Unreserved" characters are based on RFC 3986.
 
         internal static ReadOnlySpan<bool> UnreservedReservedTable => new bool[0x80]
```

Comment:
This should be reverted.

Reply:
The analyzers complained about the whitespace, therefore I deleted it.

Reply:
It's already fixed in master.